sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -315,7 +314,7 @@ public String getQueueUrl() {
      * @throws StorageException If the queue doesn't exist, a stored access policy doesn't have all fields filled out,
      * or the queue will have more than five policies.
      */
-    public Mono<Void> setAccessPolicy(List<SignedIdentifier> permissions) {
+    public Mono<Void> setAccessPolicy(Iterable<SignedIdentifier> permissions) {
```

Comment:
The code snippet link above is incorrect and will likely result in checkstyle issues and missing code samples?